Shared/AddNewLectureForm.razor

```diff
@@ -69,6 +69,13 @@
             });
         }
 
+        if (createLectureModel.SpeakerId == Session.Chair.Id)
+        {
+            errors.Add(nameof(createLectureModel.When), new (){
```

Comment:
Klucz tutaj powinien wskazywać na pole w formularzu, które spowodowało błąd, w tym przypadku `createLectureModel.SpeakerId`. Przy tym kodzie błąd wygląda tak:

![image](https://user-images.githubusercontent.com/20441980/149085679-60cbe429-068e-4eef-9257-bee79f31c9e9.png)

Pole z datą jest zaznaczone na czerwono, jakby to ono spowodowało błąd. Jeśli zmienimy to na `SpeakerId`, to wygląda tak:

![image](https://user-images.githubusercontent.com/20441980/149085837-c9a6012b-7b13-44c9-bd13-79c182f134b3.png)

Zaznaczone jest pole z wyborem Speakera.